Omikron/Factfinder/Block/FF/Communication.php

```diff
@@ -50,7 +50,7 @@ public function __construct(Template\Context $context, $data = [], Data $helper,
 
         $this->_configData = [
             'url' => [
-                'value' => '/' . Data::FRONT_NAME . '/',
+                'value' =>  ($this->_helper->isEnrichmentEnabled() ? '/' . Data::FRONT_NAME . '/' : $this->_helper->getAddress()),
```

Comment:
Can you please explain the usage of the Constant `Data::FRONT_NAME`? 
Is it also possible that you install the changes of this PR to our Magento Instance?